protocols/src/N2NP.nroff

```diff
@@ -235,17 +235,17 @@ It is described with the value 0x2 (2) in the packet type field.
 SERVICE
 .LP
 .in 2
-The SERVICE packet is used to describe a payload type, in order to the
+The SERVICE packet is used to describe a payload type in order for the
```

Comment:
The comma was necessary.
```
[...] payload type, in order [...]
                  ^
```

Reply:
The comma is not necessary, it isn't wrong to put there (and I will because it does make things clearer, especially after lots of future edits might have complicated the sentence), but omitting it is grammatically still correct as the independent phrase it at the beginning of the sentence, followed by a dependent phrase